<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IOT GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdi"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_quitter"/>
    <addaction name="action_Connection_Mosquitto"/>
    <addaction name="action_Subscribe"/>
    <addaction name="action_Stop_Mosquitto"/>
    <addaction name="action_Remove_Subscribe"/>
   </widget>
   <widget class="QMenu" name="menu_affichage">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
   </widget>
   <addaction name="menu_fichier"/>
   <addaction name="menu_affichage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>&amp;Debug</string>
   </property>
  </action>
  <action name="action_map">
   <property name="text">
    <string>&amp;Map</string>
   </property>
  </action>
  <action name="action_view_mesures">
   <property name="text">
    <string>M&amp;esures</string>
   </property>
  </action>
  <action name="action_Connection_Mosquitto">
   <property name="text">
    <string>&amp;Connection Mosquitto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Subscribe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Subscribe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Stop_Mosquitto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>S&amp;top Mosquitto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Remove_Subscribe">
   <property name="text">
    <string>&amp;Remove Subscribe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
